lib/msal-common/src/response/AuthenticationResult.ts

```diff
@@ -22,4 +23,6 @@ export class AuthenticationResult {
     extExpiresOn?: Date;
     state?: string;
     familyId?: string;
-}
+    cloudGraphHostName?: string;
+    msGraphHost?: string; 
```

Comment:
Difference between `cloudGraphHostName` and `msGraphHost` ?

Reply:
not entirely sure, both come back as separate params though so I forwarded them.

Reply:
cloudGraphHostname = AAD Graph, msGraphHost = MS Graph